quiz.js

```diff
@@ -17,8 +17,23 @@ db.dropDatabase();
 
 // your code begins here
 
+  db.houses.insert({
+  name: 'House Arryn',
```

Comment:
Don't forget to keep your naming consistent. Unlike SQL, which will complain if your data doesn't fit the mold for your table, NoSQL doesn't care what kind of data you're storing, and doesn't enforce consistency.